### PR TITLE
Add USDC image for noble chain

### PR DESCRIPTION
**Pull request by boojamya**

### CODE REVIEW

File: noble/assetlist.json [hunk 1]
```diff
@@ -45,7 +45,8 @@
       "name": "Frienzies",
       "symbol": "FRNZ",
       "logo_URIs": {
-        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/frnz.png"
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/frnz.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/frnz.svg"
```
Comment (JeremyParish69) [inline noble/assetlist.json:49]: Is there anything we can do to fix this SVG? It looks not right to me...

Comment (boojamya) [reply noble/assetlist.json:49]: Hmm good catch!
Investigating!

Comment (boojamya) [reply noble/assetlist.json:49]: If I download the image locally and view it, it seems normal. 
We'll keep this as-is for now. If we get a different version, we'll create another PR.
Thanks @JeremyParish69!

Comment (JeremyParish69) [reply noble/assetlist.json:49]: you're right! My bad. Thanks for checking!

**Pull request merged**